src/styles/colour/index.md.njk

```diff
@@ -45,70 +48,40 @@ the GOV.UK colour palette when you update GOV.UK Frontend.
         <td class="app-colour-list-column app-colour-list-column--notes">
           {{colour.notes}}
         </td>
-
       </tr>
     {% endfor %}
-    
-
   {% endfor %}
   </tbody>
 </table>
 
-## Greyscale
-
-<table class="govuk-body app-colour-list" summary="Table of greyscale colours"> 
-
-  <tbody>
-  {% for colour in colours.greyscale %}
-
-    <tr class="app-colour-list-row">
-
-      <th class="app-colour-list-column app-colour-list-column--name">
-        <span class="app-swatch {% if colour.colour == "#ffffff" %}app-swatch-border{% endif %}" style="background-color:{{colour.colour}}"></span>
-        <code>{{colour.name}}</code>
-      </th>
-
-      <td class="app-colour-list-column app-colour-list-column--colour">
-        {{colour.colour}}
-      </td>
-      <td class="app-colour-list-column app-colour-list-column--notes">
-        {{colour.notes}}
-      </td>
-
-    </tr>
+## Colour palette
 
-  {% endfor %}
-  </tbody>
-</table>
+Use these colours for graphs and supporting material.
 
-## Extended colours
+To reference colours from the palette directly you should use the `govuk-colour`
+function.
```

Comment:
Added this as function sounds a bit jargony and I'm not 100% sure all of our audience will understand it.

```suggestion
function, as shown below.
```

Reply:
From an accessibility point of view, you would normally avoid location references like 'below'.

So instead of 'as show below', could this be 'To reference colours from the palette directly, use `govuk-colour("colour name")`.

What do you think?

Reply:
Yep I'm happy with that.

Reply:
I don't think you need the word 'directly'. It made me trip up over the sentence.
```suggestion
To reference colours from the palette, use the `govuk-colour` function.
```

Reply:
That was my initial reaction too @edwardhorsford but I think it IS important because, and correct me when you're wrong, when you're using the Sass variables you are still using the palette colours, but you're doing so indirectly because we've applied a layer of context. So the colour we use could change but the context won't... so I think it's necessary to make that distinction. Is that right?